internal/rest/receipt/receiptstore.go

```diff
@@ -201,12 +198,6 @@ func (r *receiptStore) GetReceipts(res http.ResponseWriter, req *http.Request, p
 		return
 	}
 
-	res.Header().Set("Content-Type", "application/json")
```

Comment:
Why is this no longer required?

Is this explicitly done in `marshalAndReply`?

Reply:
indeed, moved to `marshalAndReply` for both this and `GetReceipt`